### ISSUE TITLE
Scrollable element is not keyboard accessible - uids.brand.uiowa.edu

### ISSUE THREAD

**Issue by briand44** [uiowa/uids#1040]
# Motivation

Siteimprove is reporting "Scrollable element is not keyboard accessible". This is a WCAG 2.1 A compliance issue.

# Siteimprove info

**Technical doc**
https://alfa.siteimprove.com/rules/sia-r84

Siteimprove link to existing issue
- https://my2.siteimprove.com/Inspector/997947/A11y/Page?pageId=24734804921&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r84/failed/UnozNW9aYXI=;MmQzYjM0ZDM=
- https://my2.siteimprove.com/Inspector/997947/A11y/Page?pageId=69552798044&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r84/failed/L3cwMVZLOTg=;MTM3YzE2ODY=

Example
- https://uids.brand.uiowa.edu
- https://uids.brand.uiowa.edu/?path=%2Fstory%2Fnot-implemented-slider--slider

**Scrollable element is not keyboard accessible**
Issue
2.1.1: Keyboard
2.1.3: Keyboard (No Exception)
This page has a scrollable section that can't be operated by keyboard.

**Comment by briand44** [comment]
## Grooming
 
**Quadrant:** 

Complicated

 **Discussion:**
 - discussed updating may solve some of the uids accessibility issues.
 - Pair with other uids issues https://github.com/uiowa/uids/issues/1039

**Proposed solution or definition of done:** 
- See if Storybook update fixes this, if not...
- investigate and understand the problem
- implement solution
- verify siteimprove is happy

**Comment by joewhitsitt** [comment]
Wonder if this is specific to the HTML viewer whitespace-se/storybook-addon-html

Update: the upgrade in https://github.com/uiowa/uids/pull/1042 does not immediately solve the issue, but it only seems to show up on components with the HTML tab present.

**Comment by joewhitsitt** [comment]
Confirmed by disabling the feature, the siteimprove doesn't report the issue.

```diff

diff --git a/.storybook/main.mjs b/.storybook/main.mjs
index 48762d9d90..7f2544c1b2 100644
--- a/.storybook/main.mjs
+++ b/.storybook/main.mjs
@@ -6,7 +6,7 @@ const config = {
   addons: [
     '@storybook/addon-docs',
     '@storybook/addon-links',
-    '@whitespace/storybook-addon-html',
+    // '@whitespace/storybook-addon-html', // temp disable to test
   ],
   features: {
     actions: false,
```

**Comment by joewhitsitt** [comment]
Found https://github.com/storybookjs/storybook/pull/33876

Going to read through the costs of upgrading 9 to 10 to get this.
